src/GraphQLParser.ApiTests/ApiApprovalTests.Public_Api_Should_Not_Change_Inadvertently.approved.txt

```diff
@@ -308,6 +317,10 @@ namespace GraphQLParser.AST
     {
         System.Collections.Generic.List<GraphQLParser.AST.GraphQLDirective>? Directives { get; set; }
     }
+    public interface IHaveDescription
```

Comment:
Let's unify this with `IHasDirectivesNode`, so `IHasDescriptionNode`.

Reply:
updated